client/src/api/homies.js

```diff
@@ -0,0 +1,15 @@
+async function addUser(user) {
+  const response = await fetch('api/homies', {
```

Comment:
Be careful with related paths. This could lead to issues when you are on a subpage.

```suggestion
  const response = await fetch('/api/homies', {
```

Reply:
Thanks for the advice, I adjusted the path.